6starter.tex

```diff
@@ -166,30 +166,30 @@
 	%  & & \\
 	
  \hline
- -1 & SD Report Start: & \Tstrut\\
-    & -1 = begin annual SD report in start year; and & \\
-    & <year> = begin SD report this year. & \Bstrut\\
+ -1 & \Gls{sd} Report Start: & \Tstrut\\
```

Comment:
I don't know if the capital Gls works here, wouldn't you want the rendered version to be "sd" not "Sd"?

Reply:
It is the first occurrence so in the pdf if show "Standard deviation (SD)"